lib/ansible/modules/monitoring/zabbix/zabbix_hostmacro.py

```diff
@@ -47,6 +47,12 @@
         required: false
         choices: ['present', 'absent']
         default: "present"
+    force:
+        description:
+            - If C(yes) don't update an existing macro
```

Comment:
Don't you mean the opposite?

Reply:
yeah I do. Do you think it would be clearer as 

`- If C(no) don't update an existing macro`
or
`- Only updates an existing macro if C(yes) `

Reply:
"Only updates an existing macro if set to C(yes)." would be good. Less negation is better :-)